src/main/resources/static/api.yml

```diff
@@ -5275,8 +5275,11 @@ components:
             arrivalDate:
               type: string
               format: date
+            summaryData:
+              $ref: '#/components/schemas/AnyValue'
           required:
             - arrivalDate
+            - summaryData
```

Comment:
Will having this as required before the frontend is ready break existing requests?

Reply:
I've modified this to be optional, and if it's not provided will default to `{}`.